features/facebookImport/src/components/advertisingValueMiniStory/advertisingValueMiniStory.css

```diff
@@ -11,8 +11,7 @@
     font-size: 20px;
 }
 .advertising-value-mini-story ul li p {
-    font-family: "Jost Medium";
-    font-weight: 800;
```

Comment:
I have a hunch we might _still_ need to set the weight, I don't know enough about how fonts work though, so could use a bit of research.

Reply:
I don't think so, tbh. I created that one with extra weight as a replacement with David because, we didn't have Jost Bold available. But the font itself (Jost Bold) is really bold, so I think it should be enough here. And according to Arne, we don't need extra weight, if the font is correctly installed. And this works at least for font-bold quite well from my point of view